### DISCUSSION TITLE
등록한 월드 상품(Item) 정보를 ZepetoIWPButton 클릭 이외의 방법으로 얻을 수 있는 방법이 있나요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#1106] [5. Q&A]
하고 싶은 것: 
- 월드 상품의 구매시 필요한 Zem의 개수를 확인하고 싶습니다:)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, IWP 상품의 가격을 미리 가져오는 API는 현재 없습니다.  
구매 완료한 아이템의 정보는 가져올 수 있습니다.
https://studio.zepeto.me/guides/linking-purchasecomplete-event1
